style.css

```diff
@@ -5,9 +5,12 @@
   box-sizing: border-box;
   margin: 0;
   padding: 0;
-  scroll-behavior: smooth;
-  overflow: hidden;
-  font-family: 'roboto', Arial, Verdana, Geneva, Tahoma, sans-serif;
+  overflow: scroll;
```

Comment:
- [ ] Avoid scrolling the text from the description, for a better user experience. 😄 
- Figma template
![image](https://user-images.githubusercontent.com/58642949/185605159-0ebabd9f-1740-4430-bcc0-1b9277c55e3b.png)

- Yours

![image](https://user-images.githubusercontent.com/58642949/185604839-0dab6bc4-d426-404f-8c48-48f7b9cfb525.png)